tests/installation/zdup.pm

```diff
@@ -40,6 +40,11 @@ sub run() {
     my $defaultrepo;
     if (get_var('SUSEMIRROR')) {
         $defaultrepo = "http://" . get_var("SUSEMIRROR");
+        #SUSEMIRROR not set, zdup from ftp source for online migration
+        if (check_var('TEST', "migration_zdup_online_sle12_ga")) {
+            my $online_repo = get_var("REPO_10");
```

Comment:
I think `REPO_10` isn't one of the "standard" vars so I would prefer to have some check if it is actually set.